test/Driver/driver-compile.swift

```diff
@@ -7,6 +7,10 @@
 // RUN: %FileCheck %s < %t.complex.txt
 // RUN: %FileCheck -check-prefix COMPLEX %s < %t.complex.txt
 
+// RUN: %swiftc_driver -driver-print-jobs -dump-ast -target x86_64-apple-macosx10.9 %s 2>&1 > %t.ast.txt
```

Comment:
Can you also add a test with `-o`? That's a simple way of checking that we're treating this as a kind of main output.